BREEZE.rst

```diff
@@ -46,409 +46,552 @@ Here is the short 10 minute video about Airflow Breeze
+The following environments are available:
 
-What happens next is the appropriate docker images are pulled, local sources are used to build local version
-of the image and you are dropped into bash shell of the airflow container -
-with all necessary dependencies started up. Note that the first run (per python) might take up to 10 minutes
-on a fast connection to start. Subsequent runs should be much faster.
+ * The ``docker`` environment (default): starts all dependencies required by a full integration test suite
+   (Postgres, Mysql, Celery, etc). This option is resource intensive so do not forget to
+   [stop environment](#stopping-the-environment) when you are finished. This option is also RAM intensive
+   and can slow down your machine.
+ * The ``kubernetes`` environment: Runs Airflow tests within a kubernetes cluster.
+ * The ``bare`` environment:  runs Airflow in the Docker without any external dependencies.
+   It only works for independent tests. You can only run it with the sqlite backend.
 
-.. code-block:: bash
 
-   ./breeze
+Cleaning Up the Environment
+---------------------------
 
-Once you enter the environment you are dropped into bash shell and you can run tests immediately.
+You may need to clean up your Docker environment occasionally. The images are quite big
+(1.5GB for both images needed for static code analysis and CI tests) and, if you often rebuild/update
+them, you may end up with some unused image data.
 
-Choosing environment
---------------------
+To clean up the Docker environment:
 
-You can choose the optional flags you need with ``breeze``
+1. `Stop Breeze <#stopping-breeze>`_ with ``./breeze --stop-environment``.
 
-You can specify for example python version to use, backend to use and environment
-for testing - you can recreate the same environments as we have in matrix builds in Travis CI.
+2. Run the ``docker system prune`` command.
 
-For example you could choose to run python 3.6 tests with mysql as backend and in docker
-environment by:
+3. Run ``docker images --all`` and ``docker ps --all`` to verify that your Docker is clean.
 
-.. code-block:: bash
+   Both commands should return an empty list of images and containers respectively.
 
-   ./breeze --python 3.6 --backend mysql --env docker
+If you run into disk space errors, consider pruning your Docker images with the ``docker system prune --all`` command. You may need to restart the Docker Engine before running this command.
 
-The choices you made are persisted in ``./.build/`` cache directory so that next time when you use the
-``breeze`` script, it will use the values that were used previously. This way you do not
-have to specify them when you run the script. You can delete the ``.build/`` directory in case you want to
-restore default settings.
+In case of disk space errors on macOS, increase the disk space available for Docker. See `Prerequsites <#prerequisites>`_ for details.
```

Comment:
```suggestion
In case of disk space errors on macOS, increase the disk space available for Docker. See `Prerequisites <#prerequisites>`_ for details.
```